core/src/main/java9/com/linecorp/armeria/internal/Java9VersionSpecific.java

```diff
@@ -36,4 +39,9 @@ public final long currentTimeMicros() {
         return TimeUnit.SECONDS.toMicros(now.getEpochSecond()) + TimeUnit.NANOSECONDS.toMicros(
                 now.getNano());
     }
+
+    @Override
+    public <T> CompletableFuture<T> newRequestContextCompletableFuture(RequestContext ctx) {
```

Comment:
How about renaming `newRequestContextCompletableFuture` -> `newRequestContextAwareCompletableFuture`?

Reply:
Question: Don't we need `final` here?

Reply:
+1 for renaming. Maybe just `newRequestContextAwareFuture()` because it's too long? We could rename the classes as well.

Reply:
@ikhoon Thanks for remembering my pattern for me! :D

@trustin Renamed the method. We already have a class named `RequestContextAwareFuture` though so left them